### PR TITLE
Add TF ViTBaseB16 model

### PR DESCRIPTION
**Pull request by mariecwhite**

### CODE REVIEW

File: benchmarks/TF/CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,20 @@ set(MINILM_L12_H384_UNCASED_INT32_SEQLEN128_MODULE
...
 )
 
+set(VIT_B16_CLASSIFICATION_FP32_MODULE
+  NAME
+    "VitB16Classification"
+  TAGS
+    "fp32"
+  SOURCE
+    # Mirror of https://tfhub.dev/sayakpaul/vit_b16_classification/1.
+    "https://storage.googleapis.com/iree-model-artifacts/vit_b16_classification_1.tar.gz"
```
Comment (pzread) [inline benchmarks/TF/CMakeLists.txt:40]: The file name needs to end with `-tf-model.tar.gz`; otherwise it won't be considered as a TF model.

**Comment by pzread** [comment]
I tested locally and it seems like the importer couldn't find the entry function.

When I generated the MiniLM model, I took the python defined TF model from https://github.com/nod-ai/SHARK/blob/main/tank/examples/MiniLM_tf/huggingface_MiniLM_gen.py#L22 and used `tf.saved_model.save` to export the model.

File: benchmarks/TF/CMakeLists.txt (hunk 1 above)
Comment (mariecwhite) [reply benchmarks/TF/CMakeLists.txt]: Thanks!

**Comment by mariecwhite** [comment]
> 

Wow this is not intuitive at all! Thanks for the tip. I did a bit more digging and it we can actually get the entry function name using this command:

```
saved_model_cli show --dir <path to saved model> --tag_set serve --signature_def serving_default --all
```

At the bottom of the output there is a list of concrete function names that can be used. Some models have '__call__' as the name and in those cases, one would need to do what you did and manually add a concrete function. Looks like this model already has a name (`serving` below) and using that as a parameter to `iree-import-tf` works.

```
  Function Name: 'serving'
    Option #1
      Callable with:
        Argument #1
          DType: dict
          Value: {'pixel_values': TensorSpec(shape=(None, None, None, None), dtype=tf.float32, name='pixel_values')
```

**Comment by mariecwhite** [comment]
This model doesn't work on CUDA so I'll go ahead and abandon this one and use a model that works on both x86 CPU and CUDA.

**Pull request closed without merging**